Description: Bump SONAME of libzfs and libzpool
Author: Shengqi Chen <harry-chen@outlook.com>
Forwarded: https://github.com/openzfs/zfs/pull/16609
Last-Update: 2024-10-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/libzfs/Makefile.am
+++ b/lib/libzfs/Makefile.am
@@ -70,7 +70,7 @@
 libzfs_la_LIBADD += -lutil -lgeom
 endif
 
-libzfs_la_LDFLAGS += -version-info 5:0:1
+libzfs_la_LDFLAGS += -version-info 6:0:0
 
 pkgconfig_DATA += %D%/libzfs.pc
 
--- a/lib/libzpool/Makefile.am
+++ b/lib/libzpool/Makefile.am
@@ -212,7 +212,7 @@
 libzpool_la_LIBADD += -lgeom
 endif
 
-libzpool_la_LDFLAGS += -version-info 5:0:0
+libzpool_la_LDFLAGS += -version-info 6:0:0
 
 if TARGET_CPU_POWERPC
 module/zfs/libzpool_la-vdev_raidz_math_powerpc_altivec.$(OBJEXT) : CFLAGS += -maltivec
